tests/throughput/compose.yaml

```diff
@@ -42,6 +43,40 @@ services:
       - MONGO_INITDB_ROOT_USERNAME=mongoadmin
       - MONGO_INITDB_ROOT_PASSWORD=mongoadminsecret
     entrypoint: ["/bin/bash", "/mongo-init.sh"]
+  rustfs:
+    image: rustfs/rustfs:latest
+    hostname: rustfs
+    restart: unless-stopped
+    expose:
+      - "9000"
+      - "9001"
+    ports:
+      - "9000:9000" # For local testing; remove in production
+      - "9001:9001" # For local testing; remove in production
```

Comment:
Note this is the throughput config. I think you want this in `docker-compose.override.yaml`, which doesn't get used in the prod deployment, only local dev.